com.unity.netcode.gameobjects/package.json

```diff
@@ -1,15 +1,9 @@
 {
-    "name": "com.unity.multiplayer.mlapi",
-    "displayName": "MLAPI Networking Library",
+    "name": "com.unity.netcode.gameobjects",
+    "displayName": "Netcode for GameObjects",
+    "description": "This the Core Unity Mid-level API that provides core SDK for multiplayer games within unity",
```

Comment:
we definitely need a better description than this :)
_(I'm planning to do a Docs & SA related pass on markdown files after this package name change)_